app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -38,45 +40,48 @@ public SearchResultsAdapter(Context context, List<Datum> datumList) {
     }
 
     @Override
-    public SearchResultHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-        return new SearchResultHolder(inflater.inflate(R.layout.search_item, parent, false));
+    public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        return new RssViewHolder(inflater.inflate(R.layout.rss_item, parent, false));
     }
 
     @Override
-    public void onBindViewHolder(final SearchResultHolder holder, int position) {
+    public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
         if (datum != null) {
             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
             holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
+            String link = "<a href=\""  +datum.getLink() + "\">" + context.getResources().getString(R.string.know_more) + "</a>";
```

Comment:
Don't display "know more". Mario has asked not to do so in susi_webchat gitter room today only. Just display the link.